include/zephyr/bluetooth/iso.h

```diff
@@ -621,10 +621,121 @@ int bt_iso_chan_disconnect(struct bt_iso_chan *chan);
  */
 int bt_iso_chan_send(struct bt_iso_chan *chan, struct net_buf *buf);
 
+struct bt_iso_unicast_tx_info {
+	/** The transport latency in us */
+	uint32_t latency;
+
+	/** The flush timeout in ms */
+	uint32_t flush_timeout;
```

Comment:
The flush timeout is typically given as a value in units of `iso_interval`, but thought it made more sense to provide the value as a unit of MS by doing `iso_interval * flush_timeout`

Reply:
If we supply the value in time, I don't think ms is sufficient unless it is a float. 7.5ms is a standard iso-interval and that cannot be expressed in units of ms - except for even FT :-)
Unit could be micro seconds.

Reply:
Actually the `iso_interval` is in units of 1.25ms  so will need to update the calculations, but regardless since `iso_interval` is in ms and the `FT` parameter of the event is in units of `iso_interval`, then the resolution of this value is also in ms.

Reply:
The units from the event is 1.25ms. Using microseconds would unfortunately require a 64-bit value to hold (max FT * max iso_interval > 2^32-1). 

Change to use unit of 1.25ms (we do that for many other interval issues).